urlextract/urlextract_core.py

```diff
@@ -65,20 +66,24 @@ class URLExtract(CacheFile):
     _ipv4_tld = ['.{}'.format(ip) for ip in range(256)]
     _ignore_list = set()
 
-    def __init__(self, extract_email=False, **kwargs):
+    def __init__(self, extract_email=False, cache_dns=True, **kwargs):
         """
         Initialize function for URLExtract class.
         Tries to get cached TLDs, if cached file does not exist it will try
         to download new list from IANA and save it to cache file.
 
         :param bool extract_email: True if we want to extract email from text.
             Disabled by default
+        :param bool cache_dns: True replaces socket DNS lookup with caching
+            equivalent provided by velocity or dnspython.
```

Comment:
Could you please remove `velocity` from this comment since it is not used.

Reply:
Sorry - I had supported velocity until I encountered https://github.com/s0md3v/velocity/issues/2 , but then removed it, and forgot to remove this mention